kits/beta-base/contracts/BetaKitBase.sol

```diff
@@ -1,24 +1,21 @@
-pragma solidity 0.4.18;
```

Comment:
They serve no purpose at the moment, but it will make sense once kits stop being hardcoded in the app. We can add them for consistency or remove it from bare-kit for the time being.

Also just checked the CLI and there should be no problem in publishing a package without a manifest (actually we should probably add a warning).